woocommerce/woocommerce-admin
# Add step wrapper/controller to onboarding profile wizard

Fixes #1882 

* Adds a step controller to allow switching between steps based on query params.
* Adds a profile wizard header.
* Adds base for plugin step.

A couple notes on this: /cc @justinshreve 
* Initially separated out the controller, but opted to add the controller logic directly to `profile-wizard/index` since it seemed mostly redundant.  Let me know if you see a good reason to separate.
* The header logo still needs to be updated to the new version.  If you can point to where I can find the png for the new one, I'll update.

### Screenshots
<img width="1249" alt="Screen Shot 2019-05-15 at 3 08 53 PM" src="https://user-images.githubusercontent.com/10561050/57755371-5ed3a480-7723-11e9-8e22-0c1c3006949d.png">

### Detailed test instructions:

1. Go to the dashboard.
2. Set `const profileWizardComplete = false;`.
3. Check that the header exists and styling is correct.
4. Add `?step=plugin` to the URL to navigate to the next step.
5. Make sure the next step is shown.
